fe/lib/components/Questionnaire/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -23,23 +22,21 @@ import { FormComponent, MutationVariableInput } from '../questionTypes';
 import { FormBuilder } from './FormBuilder/FormBuilder';
 
 interface QuestionnaireBuilderProps {
+  hirerId: string;
   graphqlInput?: MutationVariableInput;
```

Comment:
Unfortunately this is a bit messy; `graphqlInput` also has a hirer ID. Does this now ignore that ID?

Reply:
Yeah, I was thinking the caller would have to pull that out themselves. The other option would be to have disjoint prop types for `graphqlInput` and `hirerId`

Reply:
I think I made @lumishrestha suffer enough to know to avoid the potential complexity of the latter.

Would it make sense to take in just the `components` array rather than the whole input?

Reply:
How about d1718d5?

Reply:
😍